src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -141,7 +142,11 @@ def __init__(self, config: PretrainedConfig):
         self.dim = config.dim
         self.dropout = nn.Dropout(p=config.attention_dropout)
 
-        assert self.dim % self.n_heads == 0
+        # Have an even number of multi heads that divide the dimensions
+        if self.dim % self.n_heads != 0:
+            raise ValueError(
+                "self.n_heads (" + str(self.n_heads) + ") must divide self.dim (" + str(self.dim) + ") evenly"
```

Comment:
```suggestion
                f"self.n_heads: {self.n_heads} must divide self.dim: {str(self.dim)} evenly"
```
It is preferable to use f-strings ;) check for example [here](https://github.com/huggingface/transformers/blob/3c39c07f1160beb6190f635e2dab6a27e6d67596/src/transformers/models/opt/modeling_opt.py#L139)